snuba/core/initialize.py

```diff
@@ -0,0 +1,42 @@
+"""
+This file contains the initialization sequence for snuba. Anything that needs to be created before a snuba command
+is able to run is defined in this file under the `initialize` function
+
+
+All imports are hidden behind their respective functions to avoid running any code at initialization time unless explicitly
+directed
+"""
+import logging
+
+logger = logging.getLogger("snuba_core_init")
+
+
+def _load_datasets() -> None:
+    from snuba.datasets.factory import get_enabled_dataset_names
+
+    get_enabled_dataset_names()
```

Comment:
Is there a reason you're using this function instead of the `reset_dataset_factory` function?

Reply:
My understanding was that we would only use the `reset_dataset_factory` function for testing.

Reply:
nvm I'm using it now